subsys/bluetooth/audio/tbs.c

```diff
@@ -974,6 +974,8 @@ static uint8_t terminate_call(struct tbs_service_inst *inst,
 	call->index = BT_TBS_FREE_CALL_INDEX;
 	tbs_set_terminate_reason(inst, ccp->call_index, reason);
 
+	notify_calls(inst);
```

Comment:
Since `terminate_call` in contexts where `notify_calls` is already called, this won't work. 

I assume the missing notifications are coming from `bt_tbs_terminate` and `bt_tbs_remote_terminate`, and `notify_calls`  should be moved to those functions instead. Alternatively a larger refactor to ensure that `notify_calls` is always called after `terminate_call` can be done.

Reply:
Ok, I have done the changes. Thanks for your comment.